### ISSUE TITLE
Support for Nullable Types in Handler Parameters

### ISSUE THREAD

**Issue by gdau** [martinrue/Tinyweb#3]
Hi!

Great project! Really enjoying playing with it.

Are nullable types supported as handler parameters? I can't seem to get the following handler working:

```
public class HelloWorldHandler
{
    Route route = new Route("hello/{id}", new { id = new int?() });

    public IResult Get(int? id)
    {
        return Result.String("Hello World " + id);
    }
}
```

When I go to <server>/hello or /hello/5 I get the same error:

No argument was supplied for required parameter id (System.Nullable`1[System.Int32]).

It also doesn't seem to support mapping incoming parameters to properties of classes when the type is nullable.

e.g. 

```
public class HelloWorldModel
{
    public int? Id { get; set; }
}
```

Cheers,

Gav

**Comment by martinrue** [comment]
Hi Gav,

Well noticed - turns out Convert.ChangeType(), which Tinyweb uses internally, sucks :)

I'll get a fix worked out this weekend and let you know when it's resolved.

Thanks for letting me know :)

**Comment by martinrue** [comment]
The issue should now be resolved in Tinyweb 2.2.2. Please do an "update-package tinyweb" to get the latest version.
